akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteRestartedQuarantinedSpec.scala

```diff
@@ -114,6 +115,10 @@ abstract class RemoteRestartedQuarantinedSpec
           }
         }
 
+        expectMsgPF(10 seconds) {
+          case AssociationHandle.ActorSystemQuarantinedEvent(local, remote) ⇒
+        }
```

Comment:
I have added this expectation just to test that the message is received upon quarantine. Not sure if any further validation is necessary.